data/campaigns/World_Conquest/lua/map/postgeneration_utils/engine.lua

```diff
@@ -41,6 +41,10 @@ function set_terrain_impl(data)
 	end
 end
 
+
+-- Similar to multiple set_terrain calls (see below), but it first runs all the
```

Comment:
Could I ask you to use the same format as the core LuaDoc for these comments? That is, three `---` and the various `@metadata` commands.

Reply:
hmm i could, do you have a link to a sample documentation ?

Reply:
Any file in data/lua/core/ for example.

Reply:
I guess this isn't in the PR anymore, will it be committed separately?